lando_cli/cli.py

```diff
@@ -272,20 +293,26 @@ def get_new_commits(local_branch: str, base_sha: str, repo: Path) -> list[str]:
     return commits
 
 
-def get_commit_patches(commits: list[str], repo: Path) -> list[str]:
+def get_commit_patches(commits: list[str], repo: Path) -> list[bytes]:
     """Get `git format-patch` patches for each passed commit SHA."""
     patches = []
     for idx, commit in enumerate(commits):
-        patch = git_run(["format-patch", commit, "-1", "--always", "--stdout"], repo)
+        patch = git_run_bytes(
+            ["format-patch", commit, "-1", "--always", "--stdout"], repo
+        )
         patches.append(patch)
 
     return patches
 
 
-def create_add_commit_actions(patches: list[str]) -> list[dict]:
+def create_add_commit_actions(patches: list[bytes]) -> list[dict]:
     """Given an ordered list of patches, create `add-commit` actions for each."""
     return [
-        {"action": "add-commit", "content": patch, "patch_format": "git-format-patch"}
+        {
+            "action": "add-commit-base64",
+            "content": base64.encodebytes(patch),
+            "patch_format": "git-format-patch",
```

Comment:
In my PR I don't have a `patch_format` field, I just assume that the format is correct for the target SCM. In practice we will likely never use this for hg, so I thought it was okay to omit it.

I think it will need to be removed here, otherwise we will hit schema issues with